pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -671,7 +671,7 @@ func (p *Pod) validatePodGroupMetadata(r record.EventRecorder, activePods []core
 	originalQueue := jobframework.QueueName(p)
 
 	if len(activePods) < groupTotalCount {
-		errMsg := fmt.Sprintf("'%s' group total count is less than the actual number of pods in the cluster", podGroupName(p.pod))
+		errMsg := fmt.Sprintf("'%s' group has fewer runnable pods than expected", podGroupName(p.pod))
```

Comment:
Indeed, the activePods indicate runnable or succeeded pods.

https://github.com/kubernetes-sigs/kueue/blob/b29f6588702247afbdcc7c718efc6b6b86a96c56/pkg/controller/jobs/pod/pod_controller.go#L727

So, how bout this?
```suggestion
		errMsg := fmt.Sprintf("'%s' group has fewer runnable or succeeded pods than expected", podGroupName(p.pod))
```

I think that the `runnable` indicates only `Running` and `Pending`, and the `Succeded` doesn't indicate runnable.

Reply:
Yeah, but the function is only called when creating the workload, so generally there should be no succeeded / completed pods. So, saying about them to the user may raise confusion

Reply:
Being too precise could cause confusion too

/lgtm
/approve